nft_ingester/src/program_transformers/token_metadata/v1_asset.rs

```diff
@@ -41,26 +41,17 @@ pub async fn burn_v1_asset<T: ConnectionTrait + TransactionTrait>(
     id: FBPubkey,
     slot: u64,
 ) -> Result<(), IngesterError> {
-    let id = id.0;
-    let slot_i = slot as i64;
+    let (id, slot_i) = (id.0, slot as i64);
     let model = asset::ActiveModel {
         id: Set(id.to_vec()),
         slot_updated: Set(Some(slot_i)),
         burnt: Set(true),
         ..Default::default()
     };
-
-    let mut query = asset::Entity::insert(model)
-        .on_conflict(
-            OnConflict::columns([asset::Column::Id])
-                .update_columns([asset::Column::SlotUpdated, asset::Column::Burnt])
-                .to_owned(),
-        )
+    // If the asset hasn't been indexed yet, we don't do anything.
+    let query = asset::Entity::update(model)
+        .filter(asset::Column::SlotUpdated.lt(slot_i))
```

Comment:
This is something I don't fully understand about our account based updates.  Isn't it possible that a `burn` tx happens in the same slot as a `mint`?  Or really can there only be one account final state per slot so those transactions affecting the account would _have_ to be in sequential slots?

Reply:
burn_v1_asset is only called when the metadata account is closed (which is not working right now because of how geyser account closures work).

technically yes, a burn can happen in the same slot as a mint so this is valid point. ill adjust it back.

Reply:
Ok I think I need to just study the code a bit more, I think I'm missing something.

I saw you removed the filter but added back in the `WHERE excluded.slot_updated > asset.slot_updated`, so it seems like it still would still only index the burn if the slot number was greater than the mint's slot number.

And at the moment it seems like we could instead just always index the burn becuase its the only thing that would set the burnt flag to true.